app.py

```diff
@@ -25,8 +28,15 @@ def create_movies_form():
 
 @app.post('/movies')
 def create_movie():
-    # TODO: Feature 2
+    # TODO: Feature 2 - COMPLETE
     # After creating the movie in the database, we redirect to the list all movies page
+    movie_repository.create_movie(request.form.get('movie'), request.form.get('director'), request.form.get('rating'))
```

Comment:
Here, I believe 'movie' is supposed to be named title

Reply:
I called it movie in my code, which is why it says it in app.py, but it is easy to change the variable name, so I will go ahead and change it.